### PR TITLE
Improving examples

### PR DESCRIPTION
**Pull request by icarusz**
Continuing to evolve the addition of Examples to the doc site.  Starting with two suggestions Topper made on previous PR.

### CODE REVIEW

File: examples.md [hunk 1]
```diff
@@ -7,12 +7,11 @@ nav_order: 3
 
 # Example Projects
 
-## Tupelo Explorer and Wallet
+## [Tupelo Explorer and Wallet](/examples/decentracar)
```
Comment (cap10morgan) [inline examples.md]: Why is this being linked to the Decentracar example?

File: examples/iot_lock.md [hunk 2]
```diff
@@ -22,6 +22,6 @@ There are wide applications for Tupelo for IoT because it is efficient,
 fast and even includes the possibility of devices being able to check transaction
 validity when they are offline.  A small example of offline functionality
 [can be found here](https://github.com/quorumcontrol/offliner) but hop into our
-developer chat or ask for more information if you have any questions.
+[developer chat](https://t.me/joinchat/IhpojEWjbW9Y7_H81Y7rAA) or ask for more information if you have any questions.
```
Comment (tobowers) [inline examples/iot_lock.md]: ask for more information where? :)

**Review by tobowers — CHANGES_REQUESTED** [review]

File: examples.md (hunk 1 above)
Comment (icarusz) [reply examples.md]: Fixorated.

File: examples/iot_lock.md (hunk 2 above)
Comment (icarusz) [reply examples/iot_lock.md]: There is a form on most pages... I can add it to this one or just remove that reference... hmm...

Comment (icarusz) [reply examples/iot_lock.md]: Added the info form on the page.  If someone gets this far we REALLY want to know who they are so seemed like a decent choice.  May look at slightly better styling for the form but that can be later.

**Pull request merged**